src/main/webapp/js/adminSearch.es6

```diff
@@ -53,6 +53,57 @@ $(document).ready(() => {
     $('.resetGoogleIdButton').click((e) => {
         e.stopPropagation();
     });
+
+    $('.openMailEditorWithDefaults').submit(function (e) {
+        e.preventDefault();
+        const receiveremailId = $('input[name=studentEmail]').val();
```

Comment:
`receiverEmail` may be a better name as using `Id` when it is already the email itself may be confusing.